### ISSUE TITLE
IOPRP252.IMG

### ISSUE THREAD

**Issue by DCS70** [AnimMouse/POPS-binaries#1]
This file? Do we need a License to use this item? I saw someone on a YouTube Video they had this file and the Disc icon was Green? But I try to download it here and the Icon is Not Green it's Blocked out... SO do we need a License to get this? And if so where do we get it? Please let me know? Thanks!

IOPRP252.IMG file on this website for PS2???  That one does it need a License? And why doe it not have a Green Disc icon on it? Please let me know how this works?

**Comment by AnimMouse** [comment]
> Do we need a License to use this item?

No

> I saw someone on a YouTube Video they had this file and the Disc icon was Green?

Can you attach the YouTube Video here?

> SO do we need a License to get this?

This IOPRP252.IMG file has the same hash with the file required as indicated on [POPStarter](https://bitbucket.org/ShaolinAssassin/popstarter-documentation-stuff/wiki/hdd-mode), so it seems like the file is corrupted when you downloaded it, or messed up the settings on POPStarter.

**Comment by DCS70** [comment]
Yeah that was one of the problems it kept saying it was Unsafe.   I guess different YouTubers get this from different places and those have different ICONS shown on them. So that also threw me off,
I went ahead and added it last night to my PS2 anyways, Thought I'd try it and see if it did work or not, And to my Surprise it did work. So I added all my PS1 games to it.  Three Games thoe don't seem to run right and one of those won't run at all. That's a Shame but maybe someone will somehow Patch those so they work better then? (Crosses Fingers)

Thank You! for replying...  And Thank Goodness for this website... Every other place I looked they all gave me 404 ERROR pages trying to get the code from them. But this website was the only one that did work. But again it was saying the code was Unsafe had me Worried...

But Thank Goodness it does work, Thanks again.
